### PR TITLE
Map fetch mvt

### PR DESCRIPTION
**Pull request by jeroen**

### CODE REVIEW

File: R/map_fetch.R [hunk 1]
```diff
@@ -201,8 +206,9 @@ map_fetch <- function(
       year <- paste0(c(min(year), max(year)), collapse = ",")
     }
   }
-
-  query <- rgbif_compact(list(srs = srs, taxonKey = taxonKey,
+  
+  input_srs <- ifelse(grepl("mvt", format), "EPSG:3857", srs)
```
Comment (sckott) [inline R/map_fetch.R]: @jeroen problem is i think that here we're forcing srs to be 3857 if mvt is requested, but then below `srs` is used when passing to `protolite::read_mvt_sf` - so you always request 3857 from GBIF, but then protolite uses whatever the user passed in. Is that what you intended?

Comment (jeroen) [reply R/map_fetch.R]: Yes, protolite only understands `3857` input data, so we always need that. And then we ask `protolite::read_mvt_sf` to convert it to whatever output the user requested.

Comment (sckott) [reply R/map_fetch.R]: okay, i think we should then only allow `3857` when `format="mvt"`, correct?

Comment (jeroen) [reply R/map_fetch.R]: Yes only allow `"EPSG:3857"` as input data from rgbif. The output crs may be be anything.

Comment (sckott) [reply R/map_fetch.R]: ah, so the user can still set a different srs for the output, the srs that's passed to `protolite::read_mvt_sf`?

Comment (jeroen) [reply R/map_fetch.R]: Indeed, in case of mvt, the input data is always 3857 but we can transform it into the desired coordinate system after parsing it.

**Pull request merged**